core/corecontainers/include/coretypes/listobject.h

```diff
@@ -33,7 +33,7 @@ BEGIN_NAMESPACE_OPENDAQ
 
 DECLARE_OPENDAQ_INTERFACE_EX(IList, IBaseObject)
```

Comment:
I believe that the `DEFINE_INTFID` could be completely removed by replacing the interface declare macro with `DECLARE_OPENDAQ_INTEFACE`. Is this correct @Unsttopabull ?

Reply:
Yes, I thought it would be done this way.